src/frontend/src/pages/channel/index.tsx

```diff
@@ -1,5 +1,130 @@
+                statusColor='black'
+                status='ONLINE'
+              />
+
+              <div className='flex-1'>
+                <div className='text-sm font-bold'>
+                  유저유저
+                  <span className='ml-2 text-xs text-gray-400'>
+                    {formatTime(msg.createdAt.toISOString())}
+                  </span>
+                </div>
+                <div className='text-sm'>{msg.contents.text}</div>
+              </div>
+            </div>
+          ))}
+        <div ref={messagesRef} />
+      </div>
+
+      {/* 입력창 */}
+      <div className='px-4 pb-4'>
+        <div className='h-12 flex items-center bg-discord-gray-600 rounded-md'>
+          <img
+            className='ml-3'
+            width={25}
+            height={25}
+            src='/image/chat/plus.svg'
+            alt='+'
+          />
+
+          <input
+            type='text'
+            ref={inputRef}
+            placeholder='메시지 보내기'
+            className='w-full bg-transparent text-white px-3 py-2 outline-none focus-none'
+            onKeyDown={handleKeyDown}
+          />
+
+          <div className='flex gap-3 items-center'>
+            <img src='/image/chat/gift.svg' />
```

Comment:
다음처럼 한 줄로 변경하면서 alt 속성을 줄 수 있을 것 같아요!

```suggestion
            ['gift', 'gif', 'sticker', 'emoji'].map((icon, index) => <img key={index} alt={`${icon}-icon`} src={`/image/chat/${icon}.svg` />}})
```

Reply:
map 의 달인